dts/arm/nordic/nrf52810.dtsi

```diff
@@ -200,7 +200,7 @@
 &sw_pwm {
 	timer-instance = <2>;
 	channel-count = <3>;
-	clock-prescaler = <0>;
 	ppi-base = <14>;
 	gpiote-base = <0>;
+	status = "ok";
```

Comment:
Better to do this at the board level instead.

Reply:
Actually, I'm not convinced it would be better. The activation of the nRF SW PWM driver is done through Kconfig option, so this enabling here leads only to generation of `DT_` macros, which is harmless. And it's more convenient to enable the node here than in all boards that uses this SoC. Am I missing some important consequence of doing it this way?